src/net.h

```diff
@@ -880,11 +883,13 @@ class CNode
     void MaybeSetAddrName(const std::string& addrNameIn);
 };
 
-
-
-
-
 /** Return a timestamp in the future (in microseconds) for exponentially distributed events. */
 int64_t PoissonNextSend(int64_t now, int average_interval_seconds);
 
+inline std::chrono::seconds PoissonNextSend(std::chrono::seconds now, int average_interval_seconds)
```

Comment:
Why is this returning seconds and not microseconds like the existing implementation? The difference in units might confuse callers.

Reply:
theres a lot of inconsistency- the existing method takes in an interval in seconds but then returns a value in microseconds. so for the new method, I either had to #1 - maintain the inconsistency in the function signature, or #2 - be inconsistent between the return values of the two different poisson methods. since the return value is explicit with the chrono::duration types, I opted for #2

Reply:
which do you find the least confusing? 😅